src/canvas/pixelmapFeature.js

```diff
@@ -0,0 +1,34 @@
+var inherit = require('../inherit');
+var registerFeature = require('../registry').registerFeature;
+var pixelmapFeature = require('../pixelmapFeature');
+
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class pixelmapFeature
+ *
+ * @class geo.canvas.pixelmapFeature
+ * @param {Object} arg Options object
+ * @extends geo.pixelmapFeature
+ * @returns {canvas_pixelmapFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+var canvas_pixelmapFeature = function (arg) {
+  'use strict';
+
+  if (!(this instanceof canvas_pixelmapFeature)) {
+    return new canvas_pixelmapFeature(arg);
+  }
+  pixelmapFeature.call(this, arg);
+
+  var object = require('./object');
```

Comment:
In our various renderers, we have renderer-specific changes to objects, mostly to make a renderer-specific draw method.